@here/harp-mapview/lib/text/TextStyleCache.ts

```diff
@@ -49,9 +57,13 @@ const defaultTextRenderStyle = new TextRenderStyle({
     backgroundOpacity: 0.5
 });
 
+// By default text layout provides no options for placement, but single alignment.
 const defaultTextLayoutStyle = new TextLayoutStyle({
+    // TODO: Verify why default TextStyleCache layout style has different
+    // alignment settings then defaults in TextStyle.
```

Comment:
Are you gonna check this as part of the ticket? If not, please either create a task for it and put the ticket number here or remove the comment. Otherwise the code is full of forgotten TODO comments like this.

Reply:
Ok, surprisingly both defaults (in TextStyle and TextStyleCache) are originating from the same commit, which gives a strong hint that it was probably a bug/typo. So I will use defaults from TextStyle, as those are also used in tests.